sycl-fusion/common/include/Kernel.h

```diff
@@ -193,6 +177,39 @@ class Indices {
   size_t Values[Size];
 };
 
+///
+/// Model SYCL/OpenCL kernel attributes that are handled by the JIT.
+enum SYCLKernelAttributeKind { Invalid, ReqdWorkGroupSize, WorkGroupSizeHint };
```

Comment:
I'm a bit unsure about the relevant conventions here -- should this be a scoped enumeration, potentially inside of the struct below?

Reply:
I'd make this `enum class`, rename to `Kind` and move into the class below

Reply:
Done.